apps/api-extractor/src/collector/Collector.ts

```diff
@@ -511,37 +511,16 @@ export class Collector {
       this._calculateMetadataForDeclaration(astDeclaration);
     }
 
-    // Do any of the declarations have a release tag?
+    // The most public effectiveReleaseTag for all declarations
     let maxEffectiveReleaseTag: ReleaseTag = ReleaseTag.None;
 
     for (const astDeclaration of astSymbol.astDeclarations) {
       // We know we solved this above
       const declarationMetadata: DeclarationMetadata = astDeclaration.metadata as DeclarationMetadata;
       const effectiveReleaseTag: ReleaseTag = declarationMetadata.effectiveReleaseTag;
 
-      if (effectiveReleaseTag !== ReleaseTag.None) {
-        if (maxEffectiveReleaseTag !== ReleaseTag.None && maxEffectiveReleaseTag !== effectiveReleaseTag) {
-          if (!astSymbol.isExternal) { // for now, don't report errors for external code
```

Comment:
(This inadvertently prevented `maxEffectiveReleaseTag` from being calculated correctly.)